pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1232,6 +1259,11 @@ func portsUsedByLiveMigration() []string {
 	}
 }
 
+// https://istio.io/latest/docs/ops/deployment/requirements/#ports-used-by-istio
+func portsUsedByIstio() []string {
+	return []string{"15000", "15001", "15006", "15008", "15020", "15021", "15090"}
```

Comment:
You can not initialize an slice as const https://play.golang.org/p/2zZYHsqh6TW, I think that why the other port list related to migratin is a function if we do a var with it it can be changed.

Reply:
I didn't mean a const array, but an array of const strings "export each of these ports into a constant".

This seems to work: https://play.golang.org/p/vCNokvVJpjJ

Reply:
[Done](https://github.com/kubevirt/kubevirt/compare/2ae68e9c5d93b540a80e5c6b931418c2f96edd97..0c67e2ea22d9a6a185bc2c84f1b6b295a581159c)